docs/data/sdks/javascript/index.md

```diff
@@ -983,23 +983,86 @@ var exitCallback = function {
 amplitude.getInstance().init('API_KEY', 'USER_ID', { onExitPage: exitCallback });
 ```
 
-### Custom device IDs
+### Device ID lifecycle
 
-By default, device IDs are randomly generated UUIDs. You can define a custom device ID by setting it as a configuration option or by calling `setDeviceId`.
+By default Amplitude Javascript SDK uses a base64id as device id in order to have more compact string IDs. A 22-character base64id has the same range as a 128-bit UUID, which is represented as a 36 character string.
+
+On instance initialization, the SDK initializes the device ID by the following order:
+
+1. Device id in configuration on initialization
+2. Device id from url param if `configuration.deviceIdFromUrlParam` is true
```

Comment:
2. `amp_device_id` value from URL param if `configuration.deviceIdFromUrlParam` is true 
Also, link the cross-domain tracking at the end. 
Then we can remove the note.

Reply:
Fix as suggested